### ISSUE TITLE
[Enhancement] Cnf-conformance not installing with Crystal v0.36.x

### ISSUE THREAD

**Issue by iamayushdas** [cncf/cnf-testsuite#664]
**Describe the bug**

**To Reproduce**
1. while installing

**Expected behavior**

**Screenshots**
![Screenshot from 2021-03-18 16-57-01](https://user-images.githubusercontent.com/40708551/111619563-c10ffb80-880b-11eb-9d53-c0f1b32c0a45.png)

**Device (please complete the following information):**
 - Linux OS
 - Ubuntu
 - 20.04.1
 - Architecture X64
 

**Additional context**

---

NOTE: you can enable higher logging level output via the command line or env var. to help with debugging

```
# cmd line
git clone https://github.com/cncf/cnf-conformance.git
cd cnf-conformance/
shards install
crystal build src/cnf-conformance.cr

**Comment by agentpoyo** [comment]
@iamayushdas 

What version of crystal to you have installed and what was the output of `shards install` before running the `crystal build src/cnf-conformance.cr` command?

**Comment by iamayushdas** [comment]
Shards install was performed successfully.

**Comment by agentpoyo** [comment]
This isn't a bug, marking as enhancement and compatibility since we only support Crystal v0.35.1 currently and the locked in shards versions.

@wavell @denverwilliams

**Comment by iamayushdas** [comment]
Okay👍🏻

**Comment by lixuna** [comment]
FYI - Crystal lang released v1.0.0 today, see https://github.com/crystal-lang/crystal/releases/tag/1.0.0

**Comment by lixuna** [comment]
Good day, @iamayushdas!

In the CNF Test Suite release [v0.14.0](https://github.com/cncf/cnf-testsuite/releases/tag/v0.14.0), the test suite has been updated to support Crystal v1.0.0. 

Please feel free to give it a try and share any feedback.

Thanks in advance!
